### PR TITLE
[fix bug 1210492] Implement round 2 of Win10 welcome testing.

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

**Comment by alexgibson** [comment]
Nit: could improve the line wrapping for v3 headline at medium viewports?

v3

File: bedrock/firefox/templates/firefox/win10_variants/variant-2-4.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/win10-welcome.html" %}
+
+{% block page_title %}Make Firefox your default browser now{% endblock %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/win10_variants/variant-2-4.html]: This headline will be shown to the user if Firefox is already their default browser, is that ok? If memory serves me correctly, I think in the last test we worked around this by making the headline conditional.

**Comment by alexgibson** [comment]
Aside from one question re the conditional headline, this all looks good to me, r+wc :racehorse: (change being optional).

**Comment by jpetto** [comment]
Changed things around a little bit (brought back some of previous variant CSS) to improve layout and display an appropriate headline for variant 4.

File: bedrock/firefox/templates/firefox/win10_variants/variant-2-4.html [hunk 2]
```diff
@@ -4,24 +4,32 @@
...
   <header class="main-header">
-    <h1 class="not-default">Make Firefox my Default browser now</h1>
+    <h1 class="not-default">Make Firefox your Default browser now</h1>
     <h1 class="default">{{ self.page_title() }}</h1>
-    <p class="tagline">Get the privacy, speed and choice you’ve come to expect with Firefox on Windows 10.</p>
+    <p class="tagline">
+      {{ _('We’ve designed Firefox to work seamlessly with your Windows 10 update, giving
+      you the same great browsing experience you’ve come to love, with the privacy, speed
+      and choice you’ve come to expect.') }}
+      </p>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/win10_variants/variant-2-4.html]: Nit: fix indentation

File: media/css/firefox/win10-welcome-variants.less [hunk 3]
```diff
@@ -1,183 +1,34 @@
...
-            margin: 0 auto 12px;
-        }
-    }
+@import '../sandstone/lib.less';
+
+// tagline hidden for 3 of 4 test variants (with js enabled)
+.js #win10-welcome-v2-2 .firefox-not-default .tagline,
+.js #win10-welcome-v2-3 .firefox-not-default .tagline,
+.js #win10-welcome-v2-4 .firefox-not-default .tagline,
+{
```
Comment (alexgibson) [inline media/css/firefox/win10-welcome-variants.less]: Super nit: opening brace should be on the end of preceding line

**Comment by alexgibson** [comment]
Looks great! r+wc :ocean:  - care to squarsh?

**Comment by jpetto** [comment]
Nits fixed and commits squished.

**Pull request merged**